tests/tendermint/test_event_stream.py

```diff
@@ -62,3 +64,18 @@ def test_process_unknown_event():
     event_queue = Queue()
     process_event(event_queue, event, 'test_stream_id')
     assert event_queue.empty()
+
+
+@pytest.mark.asyncio
+async def test_subscribe_events(tendermint_ws_url):
+    from bigchaindb.tendermint.event_stream import subscribe_events
+    session = ClientSession()
+    ws = await session.ws_connect(tendermint_ws_url)
+    stream_id = 'bigchaindb_stream_01'
+    await subscribe_events(ws, stream_id)
+    msg = await ws.receive()
+    assert msg.data
+    msg_data_dict = json.loads(msg.data)
+    assert msg_data_dict['id'] == stream_id
+    assert msg_data_dict['jsonrpc'] == '2.0'
+    # TODO What else should be there? Right now, getting error.
```

Comment:
This seems enough for now. To make the test more rigrous a transaction could be posted to tendermint, once the transaction is `commit`ed we should assert that an update was received in the event stream for that transaction.